Makefile

```diff
@@ -14,7 +14,11 @@ export PROJ_NAME = panix
 # Makefile flags
 # prevent make from showing "entering/leaving directory" messages
 export MAKEFLAGS   += --no-print-directory
-export GIT_VERSION := "$(shell git describe --abbrev=8 --dirty --always --tags)"
+# Panix Version
+export GIT_COMMIT := "$(shell git describe --abbrev=8 --dirty --always --tags)"
+export VER_MAJOR := "0"
+export VER_MINOR := "3"
```

Comment:
Are we bumping version numbers right before, or right after release? It looks like we're currently bumping right before, but I do really like doing it immediately after.

Reply:
The current version is 0.3.0 and we'll bump it to 0.4.0 after this December release.

Reply:
Bumped.

Reply:
We're going to start bumping versions following releases.